toolchain/add.go

```diff
@@ -31,7 +30,7 @@ func Add(dir, toolchainPath string, opt *AddOpt) error {
 		return err
 	}
 
-	srclibpathEntry := strings.SplitN(srclib.Path, ":", 2)[0]
+	srclibpathEntry := filepath.SplitList(srclib.Path)[0]
```

Comment:
If you're going to do this here, it'd be more consistent to also do it above (e.g., see https://github.com/sourcegraph/srclib/pull/150/files#diff-3e59f888b6806ff36d242aaa240cb59aL33).

It's also worth noting there is a slight deviation in behavior between the two funcs (highlighted in bold):

<pre>
// SplitList splits a list of paths joined by the OS-specific ListSeparator,
// usually found in PATH or GOPATH environment variables.
<strong>// Unlike strings.Split, SplitList returns an empty slice when passed an empty string.</strong>
func SplitList(path string) []string {
	return splitList(path)
}
</pre>

I think it's likely fine, but I wanted to point it out.